src/main/java/org/b612foundation/adam/datamodel/TargetingParameters.java

```diff
@@ -6,6 +6,9 @@
 
   /** The distance from earth that should be targeted, in km. */
   private double targetDistanceFromEarth;
+  
+  /** The distance from earth that should be targeted, in km, during the first stage of targeting */
+  private double initialTargetDistanceFromEarth;
```

Comment:
Set if here to -1. This will be the default value, and you can check later if anything overwrite it.